docs/docs/validators/partial-set-security-for-validators.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 6
+---
+
+# Partial Set Security
+[Partial Set Security](../features/partial-set-security.md) allows consumer chain to join as Opt-In or Top N.
+Here, we show how a validator can opt in, opt out, or set a custom commission rate on a consumer chain, as well
+as useful queries that a validator can use to figure out which chains it has to validate, etc.
+
+## Messages
+### How to opt in to a consumer chain?
+
+:::warning
+A validator is automatically opted in to a Top N chain if the validator belongs to the top N% of the validators on the provider chain.
+:::
+
+In a Top N chain, a validator that does not belong to the top N% of the validators on the provider can still choose
+to opt in to a consumer chain. In other words, validators can opt in, in both Opt-In and Top N chains.
+
+A validator can opt in to a consumer chain by issuing the following message:
+```bash
+interchain-security-pd tx provider opt-in <consumer-chain-id> <optional consumer-pub-key>
```

Comment:
let's not emphasize that the field is optional; see a comment by Hypha about this:

> We should not be giving any recommendation about reusing keys on consumer chains (You can use the same consensus key on all consumer chains, or opt to use a different key on each consumer chain ). Instead, we should strongly recommend using different consensus keys and then maybe have a footnote on how to reuse a consensus key.
> Personally, I think we shouldn’t tell people how to reuse the consensus key at all. Why give them instructions for something we don’t want them to do?

Reply:
Good point. Removed the "optional" references.